system-addon/lib/TopStoriesFeed.jsm

```diff
@@ -272,6 +272,15 @@ this.TopStoriesFeed = class TopStoriesFeed {
       case at.NEW_TAB_REHYDRATED:
         this.maybeAddSpoc(action.meta.fromTarget);
         break;
+      case at.SECTION_OPTIONS_CHANGED:
+        if (action.data === SECTION_ID) {
+          this.uninit();
+          this.storiesLastUpdated = 0;
+          this.topicsLastUpdated = 0;
+          this.affinityLastUpdated = 0;
```

Comment:
Seems like these initial values might just want to be in `init` instead of `constructor` ?

Reply:
Yes, fixing.